### ISSUE TITLE
Component and CSS file/directory structure

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/janis#12]
Before we go too far, let's get all of our components organized in a good directory structure.

Other SASS things to think about:
- react component dir structure should probably be coherent with our sass dir structure
- package.json references flexboxgrid-sass git repo. is this risky? Not sure if pulling a package this way considers version changes
- Should we update all units to rems?
- update uswds variables with only variables we override

**Comment by ifsimicoded** [comment]
react component dir structure should probably be coherent with our sass dir structure

**Comment by ifsimicoded** [comment]
Based on comments on PR #48 
A few to do's to still wrap up:
```
|-- js
|   |-- components (I think this should be what modules currently is)
|   |-- layout (these would be page sections, ex: sidebar or footer)
|   |-- pages (these would be unique page structures, ex: homepage)
|   `-- svg (these are just stateless functional components that render xml-style svg code)
`-- css
    |-- base
    |-- helpers (extract `_helpers.scss` to here)
    |-- layout
    |-- components
    |-- pages   
    `-- vendor
```
Think about how to separate micro/macro and components/layout. eg. where should contact info go.

**Comment by ifsimicoded** [comment]
We also need to get our act together on naming convention for files. Sometimes we have SomethingPage, SomethingIndex or just Something

**Comment by ifsimicoded** [comment]
Closed in PR #75
